content/02.introduction.md

```diff
@@ -76,17 +77,62 @@ For SARS-CoV-2 it is still being investigated whether, in addition to being spre
-
-Coronaviruses are known to cause respiratory illnesses in humans through the following possible mechanisms...[Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important, etc.]
+The viral spike protein, _S_, plays a critical role in viral replication.
+The N-terminal S1 domain used for binding and the C-terminal S2 domain for fusion [@doi:10.3390/v4061011].
+The S1 domain evolves more rapidly than the S2 domain, and different coronaviruses bind to difference receptors.
+Betacoronaviruses are known to bind to CEACAM1, Neu 5,9 Ac2, and ACE2 [@doi:10.3390/v4061011].
+
+### COVID-19: Infection and Presentation
+
+#### Pathogenic Mechanisms of Coronaviruses
+
+Until more information becomes available about the pathogenesis of SARS-CoV-2 specifically, insight can be gained by examining how the human body responds to viral pathogens generally, how infection from related viruses proceeds, and how this information intersects with what is known about COVID-19.
+Following exposure to a pathogen such as a virus, pathogenesis involves three major components: entry, replication, and spread [@isbn:0-9631172-1-1 {chap. 45}]. 
+The human immune response is double-pronged, with the innate and the adaptive immune response.
+Through innate immunity, also known as humoral or antibody-mediated immunity, the body can eliminate pathogens in circulation that have not yet entered cells.
+Adaptive immunity, or the cell-mediated response, is the process by which infected cells are destroyed.
+The antibody-mediated response works to prevent or reduce entry into cells, while the cell-mediated response works to stop the pathogen's replication and spread.
+
+##### The Innate Immune Response
+
+The antibody-mediated immune response is innate and provides a way through which the host can detect and respond to a pathogen in circulation.
+This response depends on several types of leukocytes (white blood cells) that are professional phagocytic cells, which means they are able to detect and engulf an antigen, enveloping it in a phagosome.
+The conditions within the phagosome are then modulated to destroy the pathogen through fusion of the phagosome with a lysosome or granule to form a phagolysosome.
+This allows the cell to break the pathogen into its component peptides. 
+Monocytes, which include both dendritic cells and macrophages, are both phagocytic white blood cells and professional antigen-presenting cells, which meaning that they can then present the resulting peptides on the cell surface using Major Histocompatability Complex II (MHC-II) proteins.
+However, research in related viruses suggests that SARS-CoV-2 may possess some mechanisms that facilitate evasion of the antibody-mediated response.
+_In vitro_ analysis of nidoviruses including SARS-CoV suggests that these viruses can induce the development of double-membrane vesicles, preventing the dsRNA signature from being detected by the host immune system [@doi:10.1128/jvi.02501-05].
```

Comment:
What do you mean by "dsRNA siganture" here? I think I'm confused by what the innate immune system is detecting in this case?